### PR TITLE
win_virtio_serial_data_transfer_reboot:fix python3 issue

### PR DESCRIPTION
**Pull request by xiagao**
1.support python3 command.
2.fix struct.error: argument for 's' must be a bytes object
ID: 1660281
Signed-off-by: Xiaoling Gao <xiagao@redhat.com>

### CODE REVIEW

File: qemu/deps/win_serial/serial-host-send.py [hunk 1]
```diff
@@ -11,6 +11,7 @@
 
 
 def pack_message(arg):
+    arg = arg.encode('UTF-8')
```
Comment (hereischen) [inline qemu/deps/win_serial/serial-host-send.py]: Please don't change it like this.
Write a doc str for this function that only bytes is accepted in python3.

change line 37 `with open(data_file, rb) as ff:` to load the file to bytes

Comment (xiagao) [reply qemu/deps/win_serial/serial-host-send.py]: Updated, thx.

File: qemu/deps/win_serial/serial-host-send.py [hunk 2]
```diff
@@ -34,7 +34,8 @@ def main():
     data_file = sys.argv[2]
 
     ff = open(data_file)
-    arg = ff.read(65535)
+    with open(data_file, 'rb') as ff:
```
[the remark this answers is not in the document]
Comment (xiagao) [reply qemu/deps/win_serial/serial-host-send.py]: I forgot delete L36, use `with` method is better.

**Comment by xiagao** [comment]
@vivianQizhu  The test result is failed as a product bug,you can see bugzilla comment.

**Comment by vivianQizhu** [comment]
@xiagao Yes, but we still need retest latest version, the error is different after the fix.

**Comment by xiagao** [comment]
@vivianQizhu  yes, comment 8 in bugzilla is the result after the fix, if it's not fixed,the test will not go to data check steps.

**Pull request merged**